core/src/validator.rs

```diff
@@ -672,6 +705,10 @@ impl Validator {
             config.accounts_db_test_hash_calculation,
             last_full_snapshot_slot,
         );
+        info!(
+            "Chosen backends: replaying: {:?}, banking: {:?}",
```

Comment:
with strum derived `Display`, we don't need the debug prints:
```suggestion
            "Chosen backends: replaying: {}, banking: {}",
```

Reply:
nice: 45b46242fb52a93480abd0d363cae89d14d2b497